content/versions-comparison/4.0-news.json

```diff
@@ -1,55 +1,48 @@
 [
   {
-    "title":"Assets structuration with families and attributes 🖼️",
+    "title":"Asset Manager for your digital assets 🖼️",
```

Comment:
I wouldn't talk about the Asset Manager, as it is the name of the feature and for people who never heard about it before, it doesn't mean anything.
On the contrary, Assets structuration with families is a new feature introduced by the Asset Manager which can be key to encourage migration.